gtp2ogs.js

```diff
@@ -418,34 +418,84 @@ process.on('uncaughtException', function (er) {
   }
 })
 
+// console messages : for bot admins information
+
+console.log("You are using gtp2ogs version 6.0 \n - For changelog or devel updates, please visit : https://github.com/online-go/gtp2ogs/tree/devel \n - For known issues, please visit https://github.com/online-go/gtp2ogs/KNOWN-ISSUES.md");
+
+// console warnings : for bot admins usage
+
+if (argv.maxhandicap && (argv.maxhandicapranked || argv.maxhandicapunranked)) {
+    console.log("Warning: You are using --maxhandicap in combination with --maxhandicapranked or/and --maxhandicapunranked. \n Use either --maxhandicap alone , OR --maxhandicapranked with --maxhandicapunranked. \n But don't use the 3 --maxhandicap arguments at the same time. " );
+}
+
+if (argv.minhandicap && (argv.minhandicapranked || argv.minhandicapunranked)) {
+    console.log("Warning: You are using --minhandicap in combination with --minhandicapranked or/and --minhandicapunranked. \n Use either --minhandicap alone , OR --minhandicapranked with --minhandicapunranked. \n But don't use the 3 --minhandicap arguments at the same time. " );
```

Comment:
Again so many spaces that needs to be removed. I'm going to name them again, but please fix them everywhere.
`or/and` => `and/or`
`. \n Use` => `.\nUse`
`alone ,` => `alone,`
`. \n But` => `.\nBut`
`time. " );` => `time.");`